packages/create-nx-plugin/bin/create-nx-plugin.ts

```diff
@@ -103,6 +90,23 @@ function createNxPlugin(workspaceName, pluginName) {
   );
 }
 
+function updateWorkspace(workspaceName: string) {
+  console.log(`updating workspace...`);
```

Comment:
Is this `console.log` intentional? 

If so, maybe this can be more specific like.. "Setting workspace layout"? But I don't think it's necessary.